### PR TITLE
Docs: Update links, add links to latest

### PR DESCRIPTION
**Pull request by cevich**
These links do not require any login to google and can be used directly (e.g. with `curl`).

Also update the script in the upload container, to fix URIs & make `$ALSO_FILENAME` optional (for future use).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`
 * [Continuous Integration:](contrib/cirrus/README.md) [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod/master)
 * [GoDoc: ![GoDoc](https://godoc.org/github.com/containers/libpod/libpod?status.svg)](https://godoc.org/github.com/containers/libpod/libpod)
 * Automated continuous release downloads (including remote-client):
-  * Master Branch: [https://storage.cloud.google.com/libpod-master-releases/](https://storage.cloud.google.com/libpod-master-releases/)
-  * Pull-requests: [https://storage.cloud.google.com/libpod-pr-releases/](https://storage.cloud.google.com/libpod-pr-releases/)
+  * [Latest remote client for Windows](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-windows-amd64.msi)
+  * [Latest remote client for MacOS](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-darwin-amd64.zip)
+  * [Latest Snap package](https://snapcraft.io/podman)
+  * Master Branch Index: [https://storage.googleapis.com/libpod-master-releases/](https://storage.googleapis.com/libpod-master-releases/) - append filename (`<Key>`) to URL to download any file.
```
Comment (TomSweeneyRedHat) [inline README.md]: Maybe it's just me, but I grabbed the first key on the list(ibpod-master-releasesfalselibpod-master-fedora-29-amd64.tar.gz156952324178929412019-09-26T18:40:41.788Z"143c9fde4bd33b69c6eef03532a28f42"20776943libpod-master-fedora-29-), appended it to the url and hit a NoSuchKey message.  I shortened the key to the stuff before the quote in the middle, same thing.

File: README.md [hunk 2]
```diff
@@ -9,8 +9,11 @@ popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`
...
+  * Pull-request Index: [https://storage.googleapis.com/libpod-pr-releases/](https://storage.googleapis.com/libpod-pr-releases/) - each filename includes the PR number.
```
Comment (TomSweeneyRedHat) [inline README.md]: Do I need to append here too?  Looks like a raw list.

File: README.md (hunk 1 above)
Comment (cevich) [reply README.md]: Whoa, lots of extra stuff, should only be `libpod-master-fedora-29-amd64.tar.gz` as the filename.  

So complete URL would be:  `https://storage.googleapis.com/libpod-master-releases/libpod-master-fedora-29-amd64.tar.gz`.

Hmmm, maybe I can word that better...

**Comment by cevich** [comment]
> Do I need to append here too? Looks like a raw list.

TBH, I'm not thrilled with sending people to that API-page, to go trolling through XML manually.  Ideally, we'd have a page that could slurp up the XML and present a nice table of links.  Know anyone with HTML/Scripting mojo that could do it?

Let me see what I can do to make things more clear for the short-term...

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply README.md]: I just clicked on the link and thought each line was the key (nothing telling me otherwise), so I grabbed it and it didn't go well.

**Comment by cevich** [comment]
@TomSweeneyRedHat hmm, tried to fix, I still don't like it.  Maybe it would be best if I just leave those two "index" bullets off for now?  So, only include the links any idiot can click on, for a "just downloads the thing" experience.

**Comment by cevich** [comment]
@TomSweeneyRedHat nevermind, I just went ahead and clobbered the index links.  Simpler this way.  Added another commit with some minor/related fixups to the upload container image we use.

File: contrib/upldrel/entrypoint.sh [hunk 3]
```diff
@@ -17,9 +17,11 @@ gcloud_init "$GCPJSON_FILEPATH"
...
+[[ -z "$ALSO_FILENAME" ]] || \
+    gsutil cp "$FROM_FILEPATH" "gs://$BUCKET/$ALSO_FILENAME"
 
 echo "."
 echo "Release now available for download at:"
-echo "    https://storage.cloud.google.com/$BUCKET/$TO_FILENAME"
-echo "    https://storage.cloud.google.com/$BUCKET/$ALSO_FILENAME"
+echo "    https://storage.googleapis.com/$BUCKET/$TO_FILENAME"
+[[ -z "$ALSO_FILENAME" ]] || \
+    echo "    https://storage.googleapis.com/$BUCKET/$ALSO_FILENAME"
```
Comment (TomSweeneyRedHat) [inline contrib/upldrel/entrypoint.sh]: did you mean to add this file?  If so, could you update the description and possibly the title of the PR?

Comment (cevich) [reply contrib/upldrel/entrypoint.sh]: Yes, I added a new commit for it but didn't update the PR, I'll do that now.  Thanks for the reminder.

**Pull request merged**